apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/ListOrders/mocks.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import {Defs, Image, Path, Pattern, Rect, Svg, Use} from 'react-native-svg'
+
+import {OrderProps} from './mapOrders'
+
+export const getMockOrders: () => Array<OrderProps> = () => [
+  {
```

Comment:
the data transformed to Yoroi should be mocked and exported from `@yoroi/swap` the data coming from the api should be mocked and exported from on `@yoroi/openswap`, (don't need to change now, we update later after we manage the return)